recipe/meta.yaml

```diff
@@ -12,24 +12,24 @@ source:
 
 build:
   number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
-  skip: true  # [py<35]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+  skip: true  # [py<37 or s390x]
 
 requirements:
   host:
     - python
     - pip
+    - setuptools
+    - wheel
+    - flit-core
```

Comment:
Maybe we should pin `flit-core` to `3.8.0` for transparency

Reply:
Got it, changed